setup/vault-server/entrypoint.sh

```diff
@@ -73,7 +73,7 @@ vault token create \
 vault secrets enable -path=kv-v2 kv-v2
 
 # seed the kv-v2 store with an entry our web app will use
-vault kv put kv-v2/api-key apiKey=my-secret-key
+vault kv put kv-v2/api-key api-key-field=my-secret-key
```

Comment:
You could push this to an env var so it's visible in the docker-compose setup.

Reply:
That's a good idea, let me try that 👍